src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -3221,8 +3225,8 @@ private bool HasImplicitBoxingTypeParameterConversion(TypeParameterSymbol source
             }
 
             // SPEC: From T to a type parameter U, provided T depends on U
-            if ((destination.TypeKind == TypeKind.TypeParameter) &&
-                source.DependsOn((TypeParameterSymbol)destination))
+            if (destination is TypeParameterSymbol { AllowByRefLike: false } d &&
```

Comment:
>AllowByRefLike: false [](http://example.com/codeflow?start=53&length=21)

Why is `AllowByRefLike: false` required?

Reply:
> Why is ```AllowByRefLike: false``` required?

Same response as for the change on line  2875